src/signals/settings/roles/containers/RoleFormContainer/index.js

```diff
@@ -65,18 +63,10 @@ export const RoleFormContainer = ({
     });
 
     if (responseSuccess) {
-      history.push(redirectURL.pathname, redirectURL.state);
+      history.push(redirectURL);
     }
 
-  }, [history,
-    onResetResponse,
-    redirectURL.pathname,
-    redirectURL.state,
-    responseError,
-    responseSuccess,
-    roleId,
-    showGlobalNotification,
-  ]);
+  }, [history, onResetResponse, redirectURL, redirectURL.pathname, redirectURL.state, responseError, responseSuccess, roleId, showGlobalNotification]);
```

Comment:
Forgot to remove `redirectURL.state` from list of deps

Reply:
gedaan